src/containers/UserMenu.jsx

```diff
@@ -51,6 +54,13 @@ class UserMenu extends Component {
     }
   }
 
+  handleReturn = (e) => {
+    const { orgId } = this.props
+    this.props.router.push(`/app/${orgId}/todos`)
+    e.preventDefault()
```

Comment:
I think this is ok, but might want to do the preventDefault at the top to make sure it's run.